src/plugins/dashboard/public/application/dashboard_state_manager.ts

```diff
@@ -404,6 +404,15 @@ export class DashboardStateManager {
     this.stateContainer.transitions.setOption('useMargins', useMargins);
   }
 
+  public getSyncColors() {
+    // Existing dashboards that don't define this should default to true.
```

Comment:
We're not migrating existing dashboards?

Reply:
No, I plan to leave this flag in a tri-state with `undefined` meaning the user never made a choice. This allows us to create a behavior breaking migration later on (like converting all dashboards that didn't make a choice to no color syncing). We haven't decided to do so yet, but this gives us the flexibility to do it later.